modules/instrument_list/php/module.class.inc

```diff
@@ -89,6 +91,7 @@ class Module extends \Module
             $message = 'Could not find candidate information for candidate '
                 . htmlspecialchars($gets['candID']) . ' and session '
                 . htmlspecialchars($gets['sessionID']) . '.';
+            error_log($message . ' ' . $e->getMessage());
```

Comment:
is this a debugging statement or do you mean to have it permanently in the code ?

Reply:
I can't remember... It can probably be deleted.